lib/features/identity_creator/presentation/utils/identity_creator_constants.dart

```diff
@@ -0,0 +1,4 @@
+
+class IdentityCreatorConstants {
+  static const int maxSizeIdentityInlineImage = 16; // Kilobyte
```

Comment:
IMO, should be `maxKBIdentityInlineImage` to make variable clear

Reply:
> IMO, should be `maxKBIdentityInlineImage` to make variable clear

IMO, should be `maxKBSizeIdentityInlineImage` :))